Islandora-CLAW/Crayfish
# CrayFits as part of Crayfish

**GitHub Issue**: discussed in Tech Call.

# What does this Pull Request do?

Merges the CrayFits code as a directory within Crayfish. Keeps all git history.

# What's new?

CrayFits code

* Re-licenses it as MIT (formerly GPL, probably because we thought all Islandora code had to be)
* Does this change require documentation to be updated?  - hardly, though it does permit some implementation code to be simplified
* Does this change add any new dependencies? Possibly - though each microservice has its own requirements. See composer.json.
* Does this change require any other modifications to be made to the repository (i.e. Regeneration activity, etc.)?  No
* Could this change impact execution of existing code? no.

# How should this be tested?

* Set up a site to use Crayfits from Crayfish (this could be done in buildkit or on a whole VM if anyone's still setting up that way)
* Note that you still need the separate FITS webservice
* You should be able to generate FITS derivatives.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/8-x-committers
